### PR TITLE
Coreutls2

### PR DESCRIPTION
**Pull request by LordAro**
Remaking it was easier than rebasing
Ow. My hands hurt.
See #319 for previous discussion

### CODE REVIEW

File: src/Controllers/MyRadio/actionPermissions.php [hunk 1]
```diff
@@ -95,7 +97,7 @@
     for ($i = 0; $i < sizeof($data); $i++) {
         $data[$i]['del'] = [
             'display' => 'text',
-            'url' => CoreUtils::makeURL(
+            'url' => URLUtils::makeURL(
                 'Core',
```
Comment (LordAro) [inline src/Controllers/MyRadio/actionPermissions.php]: Remain as Core? Not sure where it comes from

Comment (LloydW93) [inline src/Controllers/MyRadio/actionPermissions.php]: Pretty sure that should be "MyRadio", not "Core" :/

**Comment by LloydW93** [comment]
If you're happy you've extensively tested this, I'm happy for you to merge it.

**Comment by LloydW93** [comment]
Based on IRC comments, It's @LordAro 's problem if it breaks, so I'm happy.

**Pull request merged**